### PR TITLE
feat(homer): Add pos2bed module

### PR DESCRIPTION
**Pull request by Emiller88**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/homer/pos2bed/main.nf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+def VERSION = '4.11' // Version information not provided by tool on CLI
+
+process HOMER_POS2BED {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::homer=4.11=pl526hc9558a2_3" : null)
```
Comment (FriederikeHanssen) [inline modules/homer/pos2bed/main.nf:7]: I think there was an issue when using the build numbers for conda, so they have not been used anymore

**Review by FriederikeHanssen — COMMENTED** [review]
Looks good. Just the conda thing. I can't find the slack notes about it anymore.

File: modules/homer/pos2bed/main.nf (hunk 1 above)
Comment (Emiller88) [reply modules/homer/pos2bed/main.nf:7]: So this?
```suggestion
    conda (params.enable_conda ? "bioconda::homer=4.11" : null)
```

Comment (FriederikeHanssen) [reply modules/homer/pos2bed/main.nf:7]: yes

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**